src/fparser/two/utils.py

```diff
@@ -380,9 +411,45 @@ def __new__(cls, string, parent_cls=None):
             errmsg = u"{0}: '{1}'".format(cls.__name__, string)
         raise NoMatchError(errmsg)
 
+    def get_root(self):
+        '''
+        Returns the root of the parse tree to which this node belongs.
+
+        :returns: the node at the root of the tree.
+        :rtype: :py:class:`fparser.two.utils.Base`
+
+        '''
+        parent = self
+        while parent.parent:
+            parent = parent.parent
+        return parent
+
+    def get_child(self, node_type):
```

Comment:
I think we might have discussed removing this but I think we should keep it. Obviously `get_children(node)` would become `self.children` though and I would suggest the variable `node` is changed to `child`. Having said that, I would be happy for this method to be part of a separate utils package, along with walk, as it is not a fundamental part of the tree (unlike parent, children and root).

Reply:
I've changed `node` to `child` and kept this routine as part of the utils package. The `Base.get_child()` method has been deleted.

Reply:
Great.